D2BotSoloPlay.dbj

```diff
@@ -348,7 +348,8 @@ function timer (tick) {
 			let currInGame = getTickCount() - tick;
 			let totalTime = Developer.formatTime(gameTracker.Total + currInGame);
 			let totalInGame = Developer.formatTime(gameTracker.InGame + currInGame);
-			timeStr += ("(Total: " + totalTime + ") (IG: " + totalInGame + ") (OOG: " + Developer.formatTime(gameTracker.OOG) + ")");
+			let totalDays = Developer.totalDays(gameTracker.Days + currInGame);
```

Comment:
`let totalDays = Developer.totalDays(this.GameTracker.Total + currInGame);`

just pull the info from total rather than create a whole new variable that's storing the exact same information